lib/jwt/AccessToken.js

```diff
@@ -64,48 +58,48 @@ _.extend(ChatGrant.prototype, {
     }
     return grant;
   }
-});
+}
+
 
 /**
  * @constructor
  * @param {object} options - ...
  * @param {string} options.room - The Room name or Room sid.
  */
-function VideoGrant(options) {
-  options = options || {};
-  this.room = options.room;
-}
+class VideoGrant {
```

Comment:
Is `grant = VideoGrant(opts)` no longer needed?

Reply:
Yeah, figured that converting these into proper classes would be easier for us to maintain.